api/kyverno/v2beta1/exception_type.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v2beta1
```

Comment:
Should we put this in `v1alpha1` ?

Reply:
@eddycharly - do we have any guidelines on how to maintain API versions in the short term, and a plan for switching the stored version to v2? Its a bit confusing to me at the moment.